src/App.js

```diff
@@ -1,25 +1,170 @@
+
+    //take only 6 of the items from the array
+    const finalApiArray = shuffledApiResults.slice(0, 6)
+    
+    // making a new array (ghibliMovies) with the information needed for the app (title, image, and id)
+    const ghibliMovies = []
+
+    finalApiArray.map(function (movie) {
+      // create an object for each movie, and push object into the ghibliMovies array
+      const movieData = {
+        title: movie.title,
+        image: movie.image,
+        id: movie.id,
+        matched: false
+      }
+      return (
+        ghibliMovies.push(movieData)
+      )
+    })
+
+    // cloning the ghibliMovies array so that we can create paired cards for users to match
+    const clone = JSON.parse(JSON.stringify(ghibliMovies));
+
+    // changing the #ids for the newly cloned array, so that each item has a unique #id
+    clone.map(function (item) {
+      return (
+        item.id = item.id + 99
+      )
+    })
+
+    // concatonating the two arrays together so that the 6 pairs of cards can be displayed on screen
+    const allGhibliResults = ghibliMovies.concat(clone);
+
+    // shuffling the 12 cards, so that the order is displayed at random
+    const posterCards = shuffleArray(allGhibliResults);
+
+    setFinalCardSelections(posterCards)
+
+    setMatchedCardsTracker(0)
+    setMoves(0)
+
+  } //end of handleClick function
+  
+
+  // to keep track of the number of clicks the user makes
+  function movesCounter() {
+    setMoves (moves + 1)
+  }
+
+  // getting the API calls
+  useEffect( () => {
+    axios({
+      url: 'https://ghibliapi.herokuapp.com/films', 
+    }).then((apiData) => {
+
+      if (apiData.status === 200 || apiData.statusText === 'OK') {
+        setApiResults(apiData.data)
+      } else {
+        setErrorMessage('displayError')
+      }
+    })
+  }, [])
+
+
   return (
     <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
+      <header> 
+        <img className='totoro' src="/assets/totoro.png" alt="picture of Totoro" />
+        <h1>Do you remember Ghilbi?</h1>
       </header>
+
+      <main className='wrapper'>
+        <p className='instructions'>Test your memory! Find the matching pairs by clicking on each card to reveal it.  </p>
+
+        <button onClick={handleClick}>New Game</button>
+
+        <div className='gameContainer wrapper'>
+
+          <p className={ errorMessage }>Unfortunately the website is down right now.  Please come back later to test your memory skills!  We trust that you'll remember to do so! 😉</p>
+
+          <div className={ matchedCardsTracker === 6 ? 'userWins' : 'displayNone' }>
```

Comment:
I think you can also use : null here instead of display none but idk if that's better or not